mbid_mapping/mapping/mb_release_group_cache.py

```diff
@@ -142,21 +142,36 @@ def create_json_data(self, row):
             "rels": release_group_rels
         }
 
-        recordings = []
-        for position, name, recording_mbid, length in row["recordings"] or []:
-            recordings.append({
-                "recording_mbid": recording_mbid,
-                "name": name,
-                "position": position,
-                "length": length
+        all_recordings = []
+        mediums = []
+        for medium_name, medium_position, medium_format, tracks in row["mediums"] or []:
+            recordings = []
+            for position, name, recording_mbid, length in tracks or []:
+                recordings.append({
+                    "recording_mbid": recording_mbid,
+                    "name": name,
+                    "position": position,
+                    "length": length
+                })
+
+            mediums.append({
+                "name": medium_name,
+                "position": medium_position,
+                "format": medium_format,
+                "tracks": recordings
             })
+            all_recordings.extend(recordings)
 
         return (row["release_group_mbid"],
                 artist_mbids,
                 ujson.dumps(artist),
                 ujson.dumps({"artist": artist_tags, "release_group": release_group_tags}),
                 ujson.dumps(release_group),
-                ujson.dumps({"release_mbid": str(row["recordings_release_mbid"]), "recordings": recordings}))
+                ujson.dumps({
+                    "release_mbid": str(row["recordings_release_mbid"]),
+                    "recordings": all_recordings,
```

Comment:
recordings are stored twice in release group metadata cache at the moment for backward compatibility, this can be removed as soon as this PR is deployed to production.